server/server.js

```diff
@@ -355,12 +361,43 @@ app.get(`${apiBaseUrl}/employer-sites-detail`,
     return res.json({ data: '' });
   }));
 
+app.post(`${apiBaseUrl}/update-user`,
+  keycloak.allowRolesMiddleware('ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    await validate(AccessRequestApproval, req.body);
+    await keycloak.setUserRoles(req.body.userId, req.body.role, req.body.regions);
+    await dbClient.db[collections.USERS].updateDoc(
+      {
+        keycloakId: req.body.userId,
+      },
+      {
+        sites: req.body.sites,
+      },
+    );
+
+    const user = req.hcapUserInfo;
+    logger.info({
+      action: 'update-user_post',
```

Comment:
just commenting so we remember to update this according to the endpoint update (user_patch)